### ISSUE TITLE
Impossible de continuer ma campgne

### ISSUE THREAD

**Issue by Scordralion** [JulienNigon/SpellcasterUniversity-Bug-Report#1037]
Bon, alors il s'agit de ma 4 ou 5e université de ma deuxième campagne; la fin arrive, je donne les derniers certificats (j'ai un archimage d'ailleurs, youpie !) je dois ensuite sélectionner un grimoire... je passe au dessus d'un pour savoir ce qu'il fait, je passe au dessus du deuxième pour la même raison; un clic automatique se fait (alors que je n'ai pas cliqué avec ma souris) et le jeu plante (à ce sujet, il plante beaucoup ces derniers temps alors que quand j'ai commencé à jouer il y a quelque mois, il plantait beaucoup moins)... je recharge la partie, le jeu reprend à la fin de mon université, je dois redistribuer les diplômes (sans archimage cette fois :'( ) et... ensuite le jeu n'avance plus, il freeze et je ne peux plus interagir avec rien (mais il n'est pas planté, la musique tourne toujours et je peux encore bouger la caméra), la sélection des grimoires ne se lancent pas ... je ne peux donc pas finir cette campagne ^^'
![image](https://user-images.githubusercontent.com/61381154/81568549-374b0280-939e-11ea-907a-342e3994c45a.png)

**Comment by Neelan22** [comment]
Désolée de lire ça...
Peux tu nous envoyer ta sauvegarde stp?
Pour info, quand ce bug arrive en fin de partie, les avenirs des élèves (qui ont été fait 2 fois donc) se cumulent, donc l'archimage est resté enregistré (un bug de plus ^^)
Pour trouver la sauvegarde c'est :
PC : C:\Users\user\AppData\LocalLow\Sneaky Yak Studio\Spellcaster University
Mac : ~/Library/Application Support/Sneaky Yak Studio/Spellcaster University/Saves
merci pour ton aide !

**Comment by Scordralion** [comment]
Le chemin ne marche pas mais j'ai ceci (je ne peux le mettre qu'en word, le site n'accepte pas les fichiers en fromat XML)

[saveword.docx](https://github.com/JulienNigon/SpellcasterUniversity-Bug-Report/files/4615813/saveword.docx)

**Comment by Scordralion** [comment]
Je ne sais pas si ça change quelque chose, mais j'ai dû forcer l'arrêt au gestionnaire de tâche après avoir tenté de relancer mon jeu

**Comment by Neelan22** [comment]
non, le format a tout détruit... C'est possible de l'avoir en .txt stp?

**Comment by Scordralion** [comment]
[save 2.txt](https://github.com/JulienNigon/SpellcasterUniversity-Bug-Report/files/4615985/save.2.txt)

**Comment by Neelan22** [comment]
merci !
